tests/integration/conftest.py

```diff
@@ -102,3 +106,18 @@ def _get_plugin_list_from_ip(unit_ip):
         return wordpress_client.list_plugins()
 
     return _get_plugin_list_from_ip
+
+
+@pytest.fixture
+def openstack_environment(request):
+    """Parse the openstack rc style configuration file from the --openstack-rc argument
+
+    Return a dictionary of environment variables and values.
+    """
+    rc_file = request.config.getoption("--openstack-rc")
```

Comment:
What if the parameter was not passed? Wouldn't `rc_file` be `None` in that case?

Reply:
A new commit add a default and some comments on this option, it's an error to not pass --openstack-rc in integration tests, it's okay for unit tests.